src/blog/2023/10/blueprints.md

```diff
@@ -0,0 +1,37 @@
+---
+title: What are FlowFuse Blueprints?
+subtitle: Preconfigured Node-RED Applications
+description: Explore the concept of FlowFuse Blueprints, the pre-built solutions to simplify Node-RED experience, and take a look at the first three Blueprints we've launched for manufacturing applications.
+date: 2023-10-16
+authors: ["marian-demme"]
+#image: "/blog/2023/10/images/ANDON1.png"
+tags:
+    - posts
+    - releases
+    - flowfuse
+    - changelog
```

Comment:
I don't think this is what you intent to do. Now it shows up twice in the changelog. I actually think you can remove the `src/changelog/2023/10/blueprints.md` completely.

Reply:
```suggestion
```